### PR TITLE
Feat: Add opportunity to change CupertinoTextField suffix alignment

### PR DESCRIPTION
**Pull request by ziqq**
Feat: #152482 

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] I followed the [breaking change policy] and added [Data Driven Fixes] where supported.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

### CODE REVIEW

File: bin/internal/release-candidate-branch.version [hunk 1]
```diff
@@ -0,0 +1 @@
+flutter-3.22-candidate.0
```
Comment (Piinks) [inline bin/internal/release-candidate-branch.version:1]: This does not look right for this change, can you remove it?

**Review by Piinks — COMMENTED** [review]
Hi @ziqq welcome! It looks like this PR may have had a bad merge, and pulled in some other changes. Can you take a look? I am going to mark it as a draft for now, please mark it ready for review when it is. Thanks!

File: bin/internal/release-candidate-branch.version (hunk 1 above)
Comment (ziqq) [reply bin/internal/release-candidate-branch.version:1]: OK

**Comment by ziqq** [comment]
@Piinks, Can u check please

**Review by Piinks — COMMENTED** [review]
Hey @ziqq thanks for trying to fix the merge issue. It looks like unrelated git history is still in this PR. We won't be able to land it in this state since the cla check cannot resolve.  If it is not possible to fix the git history, would you open a new PR instead? Thank you!

**Comment by ziqq** [comment]
Hello @Piinks, OK.

**Pull request closed without merging**